apps/fluent-tester/src/E2E/common/BasePage.ts

```diff
@@ -61,34 +96,33 @@ export class BasePage {
    * If this UI element is located, we know the page as loaded correctly. The UI element we look for is a Text component that contains
    * the title of the page (this._testPage returns that UI element)  */
   async isPageLoaded(): Promise<boolean> {
-    return await this._testPage.isDisplayed();
+    return (await this._testPage).isDisplayed();
   }
 
   /* Returns true if the test page's button is displayed (the button that navigates to each test page) */
   async isButtonInView(): Promise<boolean> {
-    return await this._pageButton.isDisplayed();
+    return await (await this._pageButton).isDisplayed();
   }
 
   async clickComponent(): Promise<void> {
-    await this._primaryComponent.click();
+    await (await this._primaryComponent).click();
```

Comment:
Are we sure we need these extra `await`'s? https://blog.kiprosh.com/webdriverio-elements-chaining/

Reply:
Addressed here: https://github.com/microsoft/fluentui-react-native/pull/2226#discussion_r1001000310